editor/js/libs/ui.three.js

```diff
@@ -200,7 +200,7 @@ class UITexture extends UISpan {
 
 			if ( image !== undefined && image !== null && image.width > 0 ) {
 
-				canvas.title = texture.sourceFile;
+				canvas.title = texture.sourceFile || texture.name;
```

Comment:
I have managed to fix the loading of the zip file without any changes to `ui.three.js` and `MTLLoader`. The title of the canvas can be considered as optional and ignored.

Reply:
It will show `undefined` when users hover on the texture input, expected?

Reply:
How about `canvas.title = texture.sourceFile || ''` ? this way nothing will be shown when hovering the texture input